ccm_web/client/src/utils/handleErrors.ts

```diff
@@ -140,7 +140,10 @@ const handleErrors = async (resp: Response): Promise<void> => {
     case 400:
       throw new BadRequestError(apiErrorMessage)
     case 401:
-      if (errorBody.redirect === true) redirect('/')
+      if (errorBody.redirect === true) {
+        redirect('/')
+        await new Promise(resolve => setTimeout(resolve, 1000))
```

Comment:
So this code only hit when OAuth token is revoked from canvas console?

Reply:
I would prefer a more concrete solution than adding a timeout. But I am open for discussion, if that is not possible at all. 
With that said, I spoke with @melindakraft on this she is fine if the change doesn't make it the release and worked on next phase of the project since this doesn't not disrupt user expectation.

Reply:
Okay. I'll close this PR for now. We can discuss more at a later time.